addons/gdUnit3/src/update/GdUnitUpdate.gd

```diff
@@ -105,10 +109,12 @@ func update_progress(message :String) -> void:
 	_info_progress.value += 1
 	prints("Update ..", message)
 
-func close_open_editor_scripts(plugin :EditorPlugin) -> void:
+static func close_open_editor_scripts() -> void:
+	var plugin := EditorPlugin.new()
 	var script_editor := plugin.get_editor_interface().get_script_editor()
 	prints("Close all current opened scrpts ..")
```

Comment:
```suggestion
	prints("Closing all currently opened scripts ..")
```
That way it reads like something the program is doing, instead of an instruction to the user :)

Reply:
Thanks for correcting me, my english is not so good ;)